### ISSUE TITLE
SearchBox with "WSGI search" scripts

### ISSUE THREAD

**Issue by claudiadragoste** [qgis/qwc2-demo-app#363]
I was wondering if I can use  SearchBox.jsx component  if I use "WSGI search" scripts (for the moment, it works just with Search.jsx).

**Comment by manisandro** [comment]
Yes, searchbox also supports custom search providers defined in SearchProviders.js.

**Comment by claudiadragoste** [comment]
Ok, thank you. I will give it another try. I wasn't sure if I can use the component searchbox.jsx if we don't have installed in the server the fulltext search service. Insted of it, we have this (and this only works, for now, with the Search.jsx component)

https://github.com/qgis/QGIS-Web-Client/blob/master/wsgi/search.wsgi
